variables.tf

```diff
@@ -336,6 +339,12 @@ variable "runner_gitlab_registration_token_secure_parameter_store_name" {
   default     = "gitlab-runner-registration-token"
 }
 
+variable "runner_gitlab_access_token_secure_parameter_store_name" {
```

Comment:
question: as we usually use group access tokens: How to handle the token expiration? The agent is no longer able to register. How to inform the user? Is there an option to refresh the token automatically?

Reply:
I didn't find any clue about auto refreshing tokens, but I think that GitLab sends notification when a token is about to expire.